.github/workflows/build.yml

```diff
@@ -150,7 +150,7 @@ jobs:
       - name: Install Local Node
         run: npm install @hashgraph/hedera-local
 
-      - name: Compile Code
+      - name: Build SDK
         run: ./gradlew assemble --scan
 
       - name: Compile Code - Android Example
```

Comment:
Should this `Compile Code - Android Example` be renamed to `Build Android Example` to be consistent with the rest of the name changes?

Reply:
Agree, thanks @rbarkerSL. I updated it.